spec/features/users_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :system do
+  it 'shows the username' do
+    visit('http://localhost:3000/users/1')
+    expect(page).to have_content('Lilly')
+  end
+
+  it 'displays profile picture' do
+    visit('http://localhost:3000/users/1')
+    expect(page).to have_xpath("//img[@alt='Lilly']")
+  end
+
+  it 'shows numbers of posts the user has written' do
+    visit('http://localhost:3000/users/1')
+    expect(page).to have_content('Number of posts: 4')
+  end
+
+  it 'shows the user\'s bio' do
+    visit('http://localhost:3000/users/1')
+    expect(page).to have_content('Teacher from Poland')
+  end
+
+  it 'shows the user\'s first 3 posts' do
+    visit('http://localhost:3000/users/1')
+    expect(page).to have_content('My forth post')
+    expect(page).to have_content('My third post')
+    expect(page).to have_content('My second post')
+  end
+
+  it 'has a "See all posts" button' do
+    visit('http://localhost:3000/users/1')
+    expect(page).to have_selector(:link_or_button, 'See all posts')
+  end
+
+  it 'redirects to the user/post/show page when clicking on a post' do
+    visit('http://localhost:3000/users/1')
+    expect(page).to have_link('My forth post', href: '/users/1/posts/9')
+  end
+
+  it 'redirects to users/posts index when clicking in "See all posts"' do
+    visit('http://localhost:3000/users/1')
+    expect(page).to have_link('', href: '/users/1/posts')
+  end
+end
```

Comment:
- [ ] Please kindly include tests to assert that it actually 
  - redirects to the post's show page when you click on a user's post.
  - redirects to the user's post's index page when you click on a **see all posts**.